docs/static/bundle.js

```diff
@@ -640,14 +640,36 @@ try {
 
 });
 $_mod.run("/custom-event-polyfill$0.3.0/custom-event-polyfill");
-$_mod.def("/makeup-navigation-emitter$0.0.5/util", function(require, exports, module, __filename, __dirname) { "use strict";
+$_mod.def("/makeup-navigation-emitter$0.0.5/util", function(require, exports, module, __filename, __dirname) { 'use strict';
+
+var raf = window.requestAnimationFrame || setTimeout;
```

Comment:
do we need to check for window first (for node.js use case)? I remember a related issue from Scott: https://github.com/makeup-js/makeup-keyboard-trap/pull/2

Reply:
Ah good catch. Will update PR.